### PR TITLE
Remove ${AUTOINC} from recipes

### PR DESCRIPTION
**Pull request by JoshuaWatt**
Removes explicit ${AUTOINC} from several recipes because it prevents
parsing when building with BB_NO_NETWORK="1". Instead, these recipes are
transitioned to using the devupstream class which allows the latest
versioned recipe to pull from the released archive or a git SHA1.

Finally, to make development with the upstream sources easier, a
wpe-bleeding.inc conf file has been added. This file can be included in
local.conf to automatically pull from the most recent upstream SHA1 for
select recipes.

### CODE REVIEW

File: conf/include/webkit-bleeding.inc [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+PREFERRED_VERSION_cog = "0.4.0+gitAUTOINC+%"
+PREFERRED_VERSION_libwpe = "1.4.0+gitAUTOINC+%"
+PREFERRED_VERSION_wpebackend-fdo = "1.4.0+gitAUTOINC+%"
+PREFERRED_VERSION_wpebackend-rdk = "1.20180925+gitAUTOINC+%"
+
+SRCREV_class-devupstream_pn-cog = "${AUTOREV}"
+SRCREV_class-devupstream_pn-libwpe = "${AUTOREV}"
+SRCREV_class-devupstream_pn-wpebackend-fdo = "${AUTOREV}"
+SRCREV_class-devupstream_pn-wpebackend-rdk = "${AUTOREV}"
```
Comment (philn) [inline conf/include/webkit-bleeding.inc]: Would it make sense to do this for the wpewebkit recipe as well?

Comment (JoshuaWatt) [reply conf/include/webkit-bleeding.inc]: Sure. We could make wpewebkit pull from the WebKit git mirror (git://git.webkit.org/WebKit-https.git) and then use `devupstream.bbclass` and `${AUTOREV}` just like here. Would you like me to do that?

Comment (philn) [reply conf/include/webkit-bleeding.inc]: If it's not too much trouble, yes please. I think it makes sense in the context of this PR.

**Comment by JoshuaWatt** [comment]
Ping?

**Pull request merged**